### DISCUSSION TITLE
Where did Conventions.RavenEntityName go?

### DISCUSSION THREAD

**Discussion by nbarnwell** [ravendb/ravendb#14926] [Q&A] [answered]
When getting/setting metadata for a document, we can this approach:

https://ravendb.net/docs/article-page/5.3/csharp/client-api/session/how-to/get-and-modify-entity-metadata

However, there used to be a property on the `Conventions` class called `RavenEntityName` that could be used to obtain the correct value without using a magic string, but over the years it's gone away. Is there a document alternative?

**Comment by ayende** [comment, marked as answer]
In RavenDB 4.x and upward, that is the `@collection` attribute on the `@metadata`

There is the `GetCollectionName` on `Conventions` that you can pass a type and get the string name of the collection back.
